pyvista/plotting/tools.py

```diff
@@ -461,8 +461,53 @@ def opacity_transfer_function(mapping, n_colors, interpolate=True,
 def parse_color(color, opacity=None, default_color=None):
     """Parse color into a vtk friendly rgb list.
 
+    If color is a sequence of RGBA floats, opacity will be extracted
+    from ``color`` and not the ``opacity`` parameter.
```

Comment:
If I understand this correctly (which I'm unsure of), I'd phrase this as
```suggestion
    If ``color`` is a sequence of RGBA floats, the ``opacity`` parameter
    is ignored.
```
(give or take line wrapping).

"sequence of RGBA floats" is also a bit misleading, but reading the parameter description below that should not be confusing.

Reply:
Your understanding is correct, the opacity parameter is just a default to use when it is not present in the input (and hence forces the output to be an RGBA 4-tuple instead of an RGB 3-tuple).